Docs/Configuration.tex

```diff
@@ -340,11 +340,11 @@ \subsection{Configuration Processing}\label{configuration-processing}
 
 \subsection{Configuration Structure}\label{configuration-structure}
 
-\texttt{OC\ config} is separated into following sections, which are described
-in separate sections of this document. By default it is tried to not enable
-anything and optionally provide kill switches with \texttt{Enable} property
-for \texttt{plist dict} entries. In general the configuration is written
-idiomatically to group similar actions in subsections:
+The \texttt{OC\ config} file is separated into following sections, which are described
+in separate sections of this document. By default, it attempts not to enable
+anything and optionally provides kill switches with the \texttt{Enable} property
+for \texttt{plist dict} entries. The configuration is generally written
```

Comment:
Suggest instead: "The configuration is structured so as to attempt not to enable
anything by default, and to provide kill switches via an \texttt{Enable} property
for \texttt{plist dict} entries which represent optional plugins and the like."
(The perhaps @vit9696 for final words here, "optional plugins and the like"?)

Reply:
Will revisit